MAUI/DataGrid/Styling.md

```diff
@@ -50,143 +54,167 @@ You can write custom style for the properties in SfDataGrid.DefaultStyle class u
+</ContentPage>
 {% endhighlight %}
 {% endtabs %}
 
 ## Set datagrid style from page resources
-You can write custom style for the properties in SfDataGrid.DefaultStyle class using page resource and consume the custom style as static resource to the DefaultSyle property for the required DataGrid used in that page.
+You can write custom style for the properties in `SfDataGrid.DefaultStyle` class using page resource and consume the custom style as static resource to the DefaultSyle property for the required DataGrid used in that page.
 
 {% tabs %}
 {% highlight xaml tabtitle="MainPage.xaml"%}
-<ContentPage.Resources>
-    <ResourceDictionary>
-        <syncfusion:DataGridStyle x:Key="customStyle" 
+<ContentPage xmlns:syncfusion="http://schemas.syncfusion.com/maui">
+    <ContentPage.Resources>
+        <ResourceDictionary>
+            <syncfusion:DataGridStyle x:Key="customStyle" 
                                   RowBackground="Bisque"
                                   HeaderRowBackground="Red"
                                   RowTextColor="Red"
                                   HeaderRowTextColor="White"/>
-    </ResourceDictionary>
-</ContentPage.Resources>
-<ContentPage.Content>
-    <syncfusion:SfDataGrid DefaultStyle="{StaticResource customStyle}" ItemsSource="{Binding OrderInfoCollection}" />
-</ContentPage.Content>
+        </ResourceDictionary>
+    </ContentPage.Resources>
+    <ContentPage.Content>
+        <syncfusion:SfDataGrid DefaultStyle="{StaticResource customStyle}" ItemsSource="{Binding OrderInfoCollection}" />
+    </ContentPage.Content>
+</ContentPage>
 {% endhighlight %}
 {% endtabs %}
 
 ![App or Page resource Styling in .NET MAUI DataGrid](Images/styling/maui-datagrid-app-page-resource.png)
 
 ## Implicit styling
-The appearance of DataGrid (SfDataGrid) and its inner elements can be customized by writing style of TargetType to those control. If the key is not specified, then the style will be applied to all the SfDataGrid in its scope. You can apply specific to SfDataGrid or DataRow or cell using various properties exposed.
+The appearance of DataGrid (SfDataGrid) and its inner elements can be customized by writing style of TargetType to those control. If the key is not specified, then the style will be applied to all the SfDataGrid in its scope. You can apply specific to `SfDataGrid` or [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) or [DataGridCell] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCell.html) using various properties exposed.
 
 ### Styling Record cell
-The record cells can be customized by writing style of TargetType DataGridCell. Underlying record will be the DataContext for GridCell.
+The record cells can be customized by writing style of TargetType [DataGridCell] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCell.html). Underlying record will be the DataContext for `DataGridCell`.
```

Comment:
The record cells can be customized by the writing style of TargetType [DataGridCell] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCell.html). The underlying record will be the DataContext for `DataGridCell`.